thoth/storages/graph/postgres.py

```diff
@@ -3318,13 +3341,25 @@ def get_python_package_versions_all_count(
             if python_version is not None:
                 query = query.filter(PythonPackageVersion.python_version == python_version)
 
+            if sort_by:
+                query = self._sort_by(
+                    query=query,
+                    query_name="get_python_package_versions_all_count",
```

Comment:
I think this is not needed - no need to maintain it, we can read it from backtrace.